crates/gpui/Cargo.toml

```diff
@@ -66,8 +66,8 @@ sum_tree.workspace = true
 taffy = { git = "https://github.com/DioxusLabs/taffy", rev = "1876f72bee5e376023eaa518aa7b8a34c769bd1b" }
 thiserror.workspace = true
 time.workspace = true
-tiny-skia = "0.5"
-usvg = { version = "0.14", features = [] }
+tiny-skia = "0.11"
+usvg = { version = "0.40", features = [] }
```

Comment:
Just wanted to note that resvg reexports both tiny-skia and usvg, so there is no need for us to depend on those directly and risking to get the versions out of sync by accident.

Reply:
Yes I am aware. I can gladly change it. Just thought maybe there was a reason for the dependencies to be listed independently.

Reply:
Done